### PR TITLE
Dependencies upgrade

### PR DESCRIPTION
**Pull request by UrazAkgultan**

### CODE REVIEW

File: packages/pluggableWidgets/video-player-native/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
```
Comment (Ahmad-Elsayed) [inline packages/pluggableWidgets/video-player-native/CHANGELOG.md:8]: do we need to change the widget version?

File: packages/pluggableWidgets/image-native/src/package.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <package xmlns="http://www.mendix.com/package/1.0/">
-    <clientModule name="Image" version="1.1.0" xmlns="http://www.mendix.com/clientModule/1.0/">
+    <clientModule name="Image" version="1.1.1" xmlns="http://www.mendix.com/clientModule/1.0/">
```
Comment (Ahmad-Elsayed) [inline packages/pluggableWidgets/image-native/src/package.xml:3]: should we update the package.json of the widget as well or is changing package.xml enough ?

Comment (UrazAkgultan) [reply packages/pluggableWidgets/image-native/src/package.xml:3]: Yes, we should update the package.json as well

File: packages/pluggableWidgets/video-player-native/CHANGELOG.md (hunk 1 above)
Comment (UrazAkgultan) [reply packages/pluggableWidgets/video-player-native/CHANGELOG.md:8]: I think so, because we updated a package and it should be a new version

Comment (Ahmad-Elsayed) [reply packages/pluggableWidgets/video-player-native/CHANGELOG.md:8]: it is not updated in the PR

Comment (UrazAkgultan) [reply packages/pluggableWidgets/video-player-native/CHANGELOG.md:8]: you've already updated the version to 5.0.0 for the video player, but we haven't released that version yet

Comment (UrazAkgultan) [reply packages/pluggableWidgets/video-player-native/CHANGELOG.md:8]: You can see here: https://github.com/mendix/native-widgets/pull/97/files#diff-fe6ff758cd50fbf3912828a6e368d1e1a1517fd7393be5f87ed3d2d3b49b5458

**Pull request merged**